code/drasil-example/README.md

```diff
@@ -7,36 +7,58 @@ This folder contains our working examples (implementations in Drasil of the prev
 
 **dblpend**
   - Contains the current state of the Double Pendulum case study implemented in Drasil
+  - This is a recreation of the original, manually-created version 
+    in the https://github.com/smiths/caseStudies/tree/master/CaseStudies repo.
 
 **gamephysics**
   - Contains the current state of the Game Physics case study implemented in Drasil
+  - This is a recreation of the original, manually-created version 
+    in the https://github.com/smiths/caseStudies/tree/master/CaseStudies/gamephys repo.
```

Comment:
Don't put raw links in the README, make them text links that point to the 'right' thing.

Also, don't name a directory a "repo". When you point to a sub-directory of a repo, say so.